webapp/src/components/common/Layout/Layout.module.scss

```diff
@@ -2,7 +2,7 @@
   height: 100vh;
 
   > main {
-    @apply h-full mx-auto overflow-y-scroll px-4;
```

Comment:
It has been fixed in different ticket, please remove this changes to avoid merge conflicts.